source/voice_control/thirteen-usd-voice-remote.markdown

```diff
@@ -64,18 +64,17 @@ Before you can use this device with Home Assistant, you need to install a bit of
 
 ## Controlling Home Assistant over the ATOM Echo
 
-1. Press and hold the flat button with rounded shape on your ATOM Echo.
+1. Press the flat button with rounded shape on your ATOM Echo.
    - The rectangular button on the side is the reset button. Do not press that one.
-   - The LED should light up in blue.
+   - As soon as you press the button, the LED will light up in blue.
+   - While you are speaking, the blue LED is pulsing.
+   - Once the intent has been processed, the LED lights up in green and Home Assistant confirms the action.
 2. Say a [supported voice command](/voice_control/builtin_sentences/). For example, *Turn off the light in the kitchen*.   
       - Make sure you’re using the area name exactly as you defined it in Home Assistant.
       - You can also ask a question, such as
           - *Is the front door locked?*
           - *Which lights are on in the living room?*
-3. Let go of the button.
-   - The LED should light up in green.
-   - Home Assistant will confirm the action.
-4. Your command is not supported? [Add your own commands](/integrations/conversation/).
+3. Your command is not supported? [Add your own commands](/integrations/conversation/).
```

Comment:
```suggestion
3. Your command is not supported? Add your own command using [the sentence trigger](https://www.home-assistant.io/docs/automation/trigger/#sentence-trigger).
```

(I don't really like the link to sentence trigger because it links to YAML…)

Reply:
Thank you for the catch! Made a suggestion.